pkg/apiserver/webhooks/datavolume-mutate.go

```diff
@@ -130,9 +130,12 @@ func (wh *dataVolumeMutatingWebhook) Admit(ar admissionv1.AdmissionReview) *admi
 		sourceNamespace = targetNamespace
 	}
 
-	_, err := wh.k8sClient.CoreV1().Namespaces().Get(context.TODO(), sourceNamespace, metav1.GetOptions{})
-	if err != nil {
-		return toAdmissionResponseError(err)
+	if dataVolume.GetDeletionTimestamp() == nil {
+		// No point validating source ns exists if DV is flagged for deletion
+		_, err := wh.k8sClient.CoreV1().Namespaces().Get(context.TODO(), sourceNamespace, metav1.GetOptions{})
+		if err != nil {
+			return toAdmissionResponseError(err)
+		}
```

Comment:
Maybe just return success if deletion timestamp set.  No need to bother with the rest of this function, which I think is just about setting the clone token

Reply:
Done